tests/integration/aof-race.tcl

```diff
@@ -1,6 +1,10 @@
-set defaults { appendonly {yes} appendfilename {appendonly.aof} aof-use-rdb-preamble {no} }
+set defaults { appendonly {yes} appendfilename {appendonly.aof} appenddirname {appendonlydir} aof-use-rdb-preamble {no} }
```

Comment:
Just imitate the previous style {appendonly {yes} appendfilename {appendonly.aof}

Reply:
i think we better drop both, it's long to read and maybe slightly confusing.
also since the defaults are never gonna change, even if we refer to these in the code, i'm not sure we should bother to define them.

Reply:
Need to remove all the default value override in the existing test?

Reply:
Removed